python/nano/scripts/bigdl-nano-init

```diff
@@ -300,6 +307,12 @@ if [ "${OPENMP}" -eq 1 ]; then
     export KMP_BLOCKTIME=1
 fi
 
+# Set forced core num
+if [[ ! -z "${FORCED_CORE_NUM:-}" ]]; then
+    export OMP_NUM_THREADS=$FORCED_CORE_NUM
+    export MKL_NUM_THREADS=$FORCED_CORE_NUM
```

Comment:
We might also want to set `MKL_DYNAMIC` to `FALSE`.

Some reference here: https://www.intel.com/content/www/us/en/docs/onemkl/developer-guide-linux/2023-0/mkl-dynamic.html

Reply:
Since it will only use fewer cores than omp threads, I will just ignore this variable and leave it default.